lib/index.js

```diff
@@ -111,35 +129,70 @@ const downloadFromInfoCallback = (stream, info, options) => {
       });
```

Comment:
should remove this range request header

Reply:
didn't understand 😕

Reply:
there's a `Range` header above that is added if user has given `options.range`, it's overwritten later in chunked requests. so my initial suggestion should've been to move it to the `else` statement below, to where requests that are not chunked are handled